2021/01/25 17:48:35 : Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 665, in storage_data
    getattr(self.get_source_cart(), 'finish_storage_data')()
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/basecart.py", line 4043, in finish_storage_data
    folder_upload = self.get_folder_upload(self._migration_id)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/basecart.py", line 4055, in get_folder_upload
    folder_upload = root_path + '/' + DIR_UPLOAD + '/' + get_config_ini('src', 'file', file='test.ini')
TypeError: can only concatenate str (not "NoneType") to str

2021/01/25 17:48:38 : categories_errors 8: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2099, in category_import
    if thumbnail_id['data'][0]['ID']:
IndexError: list index out of range

2021/01/25 17:48:39 : categories_errors 13: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2099, in category_import
    if thumbnail_id['data'][0]['ID']:
IndexError: list index out of range

2021/01/25 17:48:39 : categories_errors 14: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2099, in category_import
    if thumbnail_id['data'][0]['ID']:
IndexError: list index out of range

2021/01/25 17:48:39 : categories_errors 17: Traceback (most recent call last):
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/controllers/migration.py", line 1072, in migration
    import_data = fn_import(convert_data, main, ext)
  File "/Users/vuhung/Desktop/dev_train_03/v32/cartmigration/models/cart/woocommerce.py", line 2099, in category_import
    if thumbnail_id['data'][0]['ID']:
IndexError: list index out of range

